lib/node_modules/@stdlib/array/complex128/docs/types/index.d.ts

```diff
@@ -60,7 +60,7 @@ type BinaryPredicate<U> = ( this: U, value: Complex64, index: number ) => boolea
 * @param arr - array on which the method was called
 * @returns boolean indicating whether an element in an array passes a test
 */
-type TernaryPredicate<U> = ( this: U, value: Complex64, index: number, arr: Complex64Array ) => boolean;
+type TernaryPredicate<U> = ( this: U, value: Complex64, index: number, arr: Complex128Array ) => boolean;
```

Comment:
```suggestion
type TernaryPredicate<U> = ( this: U, value: Complex128, index: number, arr: Complex128Array ) => boolean;
```

Reply:
I'm surprised that this file ever passed linting, as `Complex64` has never been defined. Same thing for `Complex64Array`.

Reply:
@Jaysukh-409 `Complex64` needs to be replaced by `Complex128` here and above.

Reply:
@Planeshifter Do you have any thoughts on why TypeScript linting did not detect this error?